packages/compiler/src/init/init-template.ts

```diff
@@ -24,6 +25,9 @@ export interface InitTemplate {
    */
   description: string;
 
+  /** Minimal Compiler Support Version */
+  minimalCompilerVersion: string;
```

Comment:
```suggestion
  minimumCompilerVersion: string;
```

or should we just have a semver range here